ush/config_defaults.sh

```diff
@@ -1157,6 +1163,9 @@ MAKE_ICS_TN="make_ics"
 MAKE_LBCS_TN="make_lbcs"
 RUN_FCST_TN="run_fcst"
 RUN_POST_TN="run_post"
+GET_OBS_CCPA_TN="get_obs_ccpa"
```

Comment:
Again, can this be one (like RUN_VX_TN) called GET_OBS_TN=get_obs? It may not work that way. I would need to go back to the RUN_VX_TN task and see what we did.

Reply:
Maybe you can't remove the individual names but you could add a GET_OBS_TN?